### PR TITLE
feat: let `startRule` rerun the rule

### PR DESCRIPTION
**Pull request by Yisaer**
let startRule rerun the rule in order to re-optimize the rule, then we can generate the plan with the latest schema.

### CODE REVIEW

**Comment by Yisaer** [comment]
I think we still need to keep `registry` after craete rule to save `ruleState` in `registry` in order to let us stop rule after starting rule. @ngjaying

File: internal/server/rule_manager.go [hunk 1]
```diff
@@ -180,11 +180,30 @@ func deleteRule(name string) (result string) {
...
+}
+
+// reRunRule rerun the rule from optimize to Open the operator in order to refresh the schema
+func reRunRule(name string) error {
 	rs, ok := registry.Load(name)
 	if !ok {
 		return fmt.Errorf("Rule %s is not found in registry, please check if it is created", name)
 	} else {
-		err := rs.Start()
+		r := rs.Rule
```
Comment (ngjaying) [inline internal/server/rule_manager.go:192]: Looks like you only want to re-construct the topo of the rule. It seems not need to change the rule registry? Just like updateRule, just need to updateTopo.

**Review by ngjaying — COMMENTED** [review]
A suggestion.

File: internal/server/rule_manager.go (hunk 1 above)
Comment (Yisaer) [reply internal/server/rule_manager.go]: updated.

**Pull request merged**